conda/environments/cudf_dev_cuda11.0.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
   - pandas>=1.0,<1.4.0dev0
   - pyarrow=5.0.0=*cuda
   - fastavro>=0.22.9
+  - python-snappy>=0.6.0
```

Comment:
Is this a dependency that we'll also need to add to the Docker containers for `cudf` to build/work correctly? If so, we'll need to add it to one of the [integration repo](https://github.com/rapidsai/integration/) packages (i.e. [rapids-build-env](https://github.com/rapidsai/integration/tree/branch-21.10/conda/recipes))

Reply:
This is only required for tests.